site/content/pages/organizing/02-Before announcing/05-announce-your-event.txt

```diff
@@ -0,0 +1,21 @@
+---
+extension: html
+filter:
+- erb
+- markdown
+title: Announce your event
+---
+Once you have a time and place, you can announce your event to the world! It's great if you've had a chance to open your CFP and registration, though those aren't essential to announce your dates.
```

Comment:
While it's fine to have a "coming soon" presence in the "upcoming" section, for inclusion on the map I think we need a specific time and place.